warehouse/query-core/src/main/java/datawave/query/index/lookup/Intersection.java

```diff
@@ -171,25 +164,12 @@ public Intersection(Collection<? extends IndexStream> streams, UidIntersector ui
         if (absent) {
             this.context = StreamContext.ABSENT;
             this.contextDebug = "found absent child";
-        } else if (areAllChildrenSameContext(streams, StreamContext.DELAYED_FIELD)) {
-            this.context = StreamContext.DELAYED_FIELD;
+        } else if (areAllChildrenSameContext(streams, StreamContext.DELAYED)) {
+            this.context = StreamContext.DELAYED;
             this.contextDebug = "delayed field";
-        } else if (areAllChildrenSameContext(streams, StreamContext.UNINDEXED)) {
-            this.context = StreamContext.UNINDEXED;
-            this.contextDebug = "all children unindexed";
-        } else if (areAllChildrenSameContext(streams, StreamContext.IGNORED)) {
-            this.context = StreamContext.IGNORED;
-            this.contextDebug = "all children ignored";
-        } else if (areAllChildrenSameContext(streams, StreamContext.EXCEEDED_TERM_THRESHOLD)) {
-            this.context = StreamContext.EXCEEDED_TERM_THRESHOLD;
-            this.contextDebug = "all children exceeded term threshold";
-        } else if (areAllChildrenSameContext(streams, StreamContext.EXCEEDED_VALUE_THRESHOLD)) {
-            this.context = StreamContext.EXCEEDED_VALUE_THRESHOLD;
-            this.contextDebug = "all children exceeded value threshold";
-            next(); // call next to populate the top key. this is a stream of days generated by the range stream
         } else if (this.children.isEmpty() && !delayedNodes.isEmpty()) {
             // we have a mix of delayed marker nodes
-            this.context = StreamContext.DELAYED_FIELD;
+            this.context = StreamContext.DELAYED;
```

Comment:
if we intersect a delay and a non-delay as long as there is at least one PRESENT or VARIABLE I think for the sake of intersection is actually PRESENT/VARIABLE?

Reply:
That is correct, but in this case the set of executable terms is empty leaving only delayed terms, thus the delay. I'll update the comment to be correct. The comment was trying to say "all terms are delayed, but none of them are of a single type so I cannot surface the most specific context". If that sounds confusing, well, this is why we're simplifying things.